### PR TITLE
Improved GetInvolved section

### PR DESCRIPTION
**Pull request by Shruti9520**

### CODE REVIEW

File: partials/tabs/getinvolved.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 
         <div class="col-md-8 col-md-offset-2">
-            <div class="title-get-involved" >get involved</div>
+            <div class="title-get-involved" >Get Involved</div>
```
Comment (hemangsk) [inline partials/tabs/getinvolved.html]: Tried the changes locally, Actually its a part of style philosophy followed throughout the website that headings are in lowercase (just like coala is always spelled with a lowercase c). So we can leave the headings at it is.

Comment (Shruti9520) [inline partials/tabs/getinvolved.html]: So, Can I again change it to lowercase?

Comment (hemangsk) [inline partials/tabs/getinvolved.html]: Ya, please do, that will help it to keep consistent

**Comment by hemangsk** [comment]
@Shruti9520 a quite important readability issue you must have noticed that the description text 
```
Take a look at newcomer.. 
```
is center aligned, which makes it a bit difficult to read.
 Its div uses description class because earlier it was just a one line sentence. But now its a collection of new line separated sentences. So you can align this description in a manner that it increase readability, maybe 'left' align or 'justify' align

**Comment by Shruti9520** [comment]
@hemangsk Can I make the three different lines in three diff grids OR should I just left align all the three lines?

**Comment by gkrishnan724** [comment]
@Shruti9520 Follow the commit message guidelines please http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html   :)

**Comment by Shruti9520** [comment]
@gkrishnan724 Thanks :)

**Comment by hemangsk** [comment]
Ya the Left align solution sounds better 👍

**Pull request closed without merging**